spin-observatory-plugin-deck/src/components/PluginContainer.tsx

```diff
@@ -6,7 +6,7 @@ import { ReactSelectInput, useDataSource } from '@spinnaker/core';
 
 import { DatePicker } from './date-picker/date-picker';
 import { ParameterSelect } from './parameters';
-import { PipelineExecutions, STATUSES } from './pipelines';
+import { IDateFilter, PipelineExecutions, STATUSES } from './pipelines';
```

Comment:
I think it makes more sense for this to be called something like `IDateRange` and then call the prop in `PipelineExecutions.tsx` `dateRange` instead, also think it's more logical for it to come from the date picker component instead of pipelines

Reply:
Updated name and moved to date picker component.